### ISSUE TITLE
Fotos

### ISSUE THREAD

**Issue by Pauletta** [Jugendhackt/Handbuch-Jugend-Hackathons#9]
Runde Fotos ähnlich dem gelayouteten PDF einfügen und von Text umfließen lassen: https://jugendhackt.github.io/Handbuch-Jugend-Hackathons/appendix/00%20Hanbuch_Jugend-Hackathons.pdf

**Comment by rhotep** [comment]
Das geht leider nicht einfach. Jedenfalls nicht so, dass es für alle (zuküftigen)Bilder und alle Browser funktioniert. Für einzelne Bilder ließe sich da eventuell was machen, aber das macht es dann allen anderen schwer, die andere Inhalte einbringen wollen. (Zum Beispiel, wenn eine Übesertzung ein anders Bild benutzen möchte oder der übersetzte Text anders umbricht)

Achtung übrigens: Der Link aufs PDF hat einen Rechtschreib Fehler; bei "Handbuch" fehlt das "d". Ich hab shcon verschiedene Links dahin gefunden, deshalb hab ich mal eine Kopie erstellt, so dass es einen mit und einen link ohne 'd' gibt.

**Comment by Pauletta** [comment]
Ja, danke! Hatte die Korrektur gesehen :) - Zu den Bildern. Was würdest du vorschlagen? Gibt es eine Lösung, die auch bei Übersetzungen funktionieren würde?
